homeassistant/components/daikin_madoka/sensor.py

```diff
@@ -0,0 +1,105 @@
+# pylint: skip-file
+"""Support for Daikin Madoka HVAC sensors."""
```

Comment:
Limit the addition of a new integration to a single platform. Either add climate or sensor, not both.

Reply:
Sorry for my lack of knowledge.. Does that mean one PR per platform even if they belong to the same component?

Reply:
Currently, this integration provides 2 platforms: `climate` and `sensor`. It should only provide 1 of those in the initial PR. More platforms can be added in subsequently PRs.

The reason is that we try to keep PRs as small as possible, in order to be able to move forward quicker.